src/home.js

```diff
@@ -0,0 +1,37 @@
+const displayPizza = document.querySelector('.display-pizza');
+
+const getPizza = async () => {
+  const url = 'https://forkify-api.herokuapp.com/api/v2/recipes?search=pizza&key=c9510895-3e98-4f84-a4ff-c56d29bdf5e0';
```

Comment:
this variable, please create it outside the function, because we will need it further in the process.

as well, the key for the API, save it in a variable outside the function, because we will all need it

Reply:
and after you created the variables outside, just use them in your function.

if in doubt, please get in contact